languages/python/exercises/concept/none/.docs/introduction.md

```diff
@@ -0,0 +1,26 @@
+When you are beginning to code, you always assign a variable to a value. Say,
+
+```python
+a = 3
+b = True
+```
+
+This declares the variable as well as defines them. But when you want to just declare the variable and not define them at start you use a special built in called as `None`.
+
```

Comment:
For the `introduction.md`, the goal is to focus only on the information needed to explain/define the concept and complete the exercise.  It should be as simple & direct as possible, with example usage.  So I think we can remove these paragraphs about variable declaration, and then go into more detail or nuance in the `after.md`.

Reply:
Noted. I will refer the docs from other Sources you have given me and alter the introduction.md. 

Although, I should reference them in the design.md ( as well as after.md ) right?

Reply:
**Correct!**  we definitely **do** want those additional resources available to the student in places like `hints.md` and `after.md`.  And we also want them in the `design.md`, so that we internally have  a record of what resources we used to build the exercise (_and what we might need to alter or update at a later time_), and what our thinking was around the concept/topic.